docs/en_us/developers/source/accessibility.rst

```diff
@@ -0,0 +1,547 @@
+  examples of exceptions.
+  
+    * If yes, and if the same text appears adjacent to or near the image in the
+      DOM, use a "NULL" value in the alt attribute, otherwise a screen reader is
+      exposed to the same content twice.
+    
+    * If yes, and if the text within the image is there simply for visual effect
+      (such as a skewed screenshot of computer code), use a "NULL" value in the
+      alt attribute.
+	  
+* Does your image contribute meaning to the current page or context?
+  
+    * If yes, and if the image is a simple graphic or photograph, the alt
+      attribute should briefly describe the image in a way that conveys the same
+      meaning that a sighted person would obtain from viewing the image. Context
+      is important. A detailed description of a photograph is rarely useful to
+      the user, unless it is in the context of a photography or art class.
+	  
+    * If yes, and if the image is a graph or complex piece of information,
+      include the information contained in the image elsewhere on the page. The
+      alt attribute value should give a general description of the complex image. You
+      can programmatically link the image with the detailed information using ``aria-
+      describedby``.
+	
+A very pragmatic guide on providing useful text alternatives is included in the
+`HTML5 specification (4.7.1.1) <http://www.w3.org/TR/html5/embedded-
+content-0.html#alt>`_ and includes a variety of example images and appropriate
+text alternatives.
+
+
+.. _Choose colors that meet minimum contrast ratios:
+
+**********************************************************
+Choose colors that meet WCAG 2.0's minimum contrast ratios
+**********************************************************
+
+A minimum contrast between foreground and background colors is critical for
+users with impaired vision. You can `check color contrast ratios
+<https://leaverou.github.io/contrast-ratio/>`_ using any number of tools
+available free online.
+
+
+.. _Test your code for accessibility:
+
+********************************
+Test your code for accessibility
+********************************
+
+The only way to determine if your feature is fully accessible is to manually
+test it using assistive technology; however, there are a number of automated
+tools you can use to perform a self assessment. Automated tools might report
+false positives and might not catch every possible error, but they are a quick
+and easy way to detect the most common mistakes.
+
+These are some automated tools for accessibility testing.
+
+* `WAVE Accessibility Toolbar <http://wave.webaim.org/toolbar/>`_
+  (Chrome/Firefox)
+  
+* `Web Developer Toolbar <https://addons.mozilla.org/en-US/firefox/addon/web-
+  developer/>`_ (Firefox)
+  
+* `Chrome Accessibility Developer Tools
+  <https://chrome.google.com/webstore/detail /accessibility-
+  developer-t/fpkknkljclfencbdbgkenhalefipecmb>`_
+
+* Your keyboard
```

Comment:
@cptvitamin Is there additional information we can add to the "Your keyboard" list item? Something to indicate how to perform testing using your keyboard, some reference URL?